packs/redis/config.yml

```diff
@@ -7,9 +7,26 @@ name: redis
 title: Redis
 
 # Description of the pack
-description: |
-  Official New Relic pack for Redis
-  Use this pack together with the New Relic Redis On Host Integration to get insight into the performance of your Redis instances.
+description: |+
+  ## A Complete Redis Monitoring System
+  ## Monitoring Redis
```

Comment:
don't we need an additional newline after the h2, before the paragraph? (for all h2's in this PR actually)
I believe that's how I have been using it, otherwise I think it will not render the paragraph separately, but take it as part of the h2. I could be wrong though...

Reply:
@jpvajda Can you confirm this?